README.rst

```diff
@@ -97,6 +97,25 @@ patched and that you don't have to worry about the details.
 API
 ===
 
+``replace(func, new_source)``
```

Comment:
rather presumptious to put it above the pre-existing API, please put it right at the bottom after `temp_patch` since I think use of this should be discouraged